Specs/DateTime/German/DatePeriodExtractor.json

```diff
@@ -564,13 +624,13 @@
     ]
   },
   {
-    "Input": "Ich würde 2016-05 nicht hier",
+    "Input": "Ich war im 05.2016 nicht hier",
```

Comment:
Same. Here for example, my understanding is that government docs use YYYY-MM-DD and YYYY-MM, don't they?

Reply:
Not that I am aware of, but I added a spec for 2016-05 back in. But the sentence was still incorrectly translated and was grammatically incorrect. I added a correct sentece with 2016-05 now.